setup.py

```diff
@@ -49,10 +49,12 @@ def read_content(filepath):
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
 
+
 def get_requirements():
     with open("requirements.txt") as f:
         return f.read().splitlines()
 
+
```

Comment:
added by tox

Reply:
I don't think this project uses Python black.

Reply:
it does. it is in tox.ini 
```[testenv:pycodestyle]
deps=
    {[py]deps}
    pycodestyle
    black
basepython = python3.7```

Reply:
Ah, I missed that. Sorry